homeassistant/components/tedee/strings.json

```diff
@@ -62,6 +62,16 @@
     "sensor": {
       "pullspring_duration": {
         "name": "Pullspring duration"
+      },
+      "door_state": {
+        "name": "Door state",
+        "state": {
+          "not_paired": "Not paired",
```

Comment:
this is not a valid state, right? because `supported_fn` returns false for NOT_PAIRED